src/Accounts/Authentication/Authentication.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```

Comment:
Why add new package reference here?

Reply:
This was automatically added after updating the SecurityCenter nuget

Reply:
This was automatically added after updating the SecurityCenter nuget. Removed.